index.bs

```diff
@@ -123,7 +123,7 @@ spec: WHATWG HTML; urlPrefix: https://html.spec.whatwg.org/
 
 spec: FIDO-CTAP; urlPrefix: https://fidoalliance.org/specs/fido-v2.0-ps-20170927/fido-client-to-authenticator-protocol-v2.0-ps-20170927.html
     type: dfn
-        text: CTAP2 canonical CBOR encoding form; url: ctap2-canonical-cbor-encoding-form
+        text: CTAP2 canonical CBOR encoding form; url: message-encoding
```

Comment:
the "ctap2-canonical-cbor-encoding-form" URL frag will be correct once the latest CTAP spec is published, whereupon the urlPrefix will be incorrect :-/

the fido-client-to-authenticator-protocol-v2.0-ps-20170927.html#message-encoding section does not correctly define "CTAP2 canonical CBOR encoding form" :(

It seems there is not good solution here until there the revised CTAP spec containing the \<dfn> for "CTAP2 canonical CBOR encoding form" is published.

Reply:
Ah, I see. I'll revert this one, then. Good catch!